benchmark/net/net-c2s-cork.js

```diff
@@ -0,0 +1,96 @@
+// test the speed of .pipe() with sockets
+
+var common = require('../common.js');
+var PORT = common.PORT;
+
+var bench = common.createBenchmark(main, {
+  len: [4, 8, 16, 32, 64, 128, 512, 1024],
+  type: ['buf'],
```

Comment:
The reason is because it can be tuned passing arguments. I have disabled them because they do not really provide more data for this benchmark. I can get them removed completely, or re-enable them in the bench.